### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by MPDADDY**
- [ ] Added media query to make the desktop version look good.
- [ ] Downloaded desktop images and added them to the desktop version of the project.
- [ ] Customised the author's section in the readme file.
- [ ] Removed the Note section in the readme as is not part of the readme file.
- [ ] Done with the optional changes suggested by the previous reviewer.
- [ ] Done with the changes to make the project look similar to the Figma template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,21 +18,38 @@
...
+                        <a href="#">contact</a>
+                    </li>
+                </ul>
+                <div class="humberger bars">
+                    <i class="fa fa-bars"></i>
+                </div>
+            </nav>
+        </div>    
     </header>
     <section class="intro">
```
Comment (divinecharlotte) [inline index.html:43]: - [ ] kindly consider making sure that the background image for this section covers the whole section as given in the Figma design  to make it an exact duplicate of the given design check the images below  for more clarification 
- your design 👇 
Screen Shot 2023-05-12 at 15 11 19


- Figma design 👇 
Screen Shot 2023-05-12 at 15 13 40

File: index.html [hunk 2]
```diff
@@ -47,36 +64,43 @@ <h4>LET'S CONNECT</h4>
...
-            <div>
+            <div class="width">
+                <h2 class="colorh2">Tonic</h2>
                 <div class="canopy">
                     <h4 class="colorh4">CANOPY</h4>
                     <ul>
-                        <li>
+                        <li class="canopylist">
                             <span>
                                 <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
```
Comment (divinecharlotte) [inline index.html:95]: - [ ] kindly consider giving your icons some border-radius and the right background color  to make them look exactly like the one provided in the design you may also download them from Figma to make it quick  and doing so will demonstrate your skills in implementing the provided design exactly check the images below to easily see the difference 
- [ ] kindly consider also giving the `see project` button the same color as the one given in Figma to make it an exact duplicate of the given one 
- your design 👇 
Screen Shot 2023-05-12 at 15 25 10


- Figma design 👇 
Screen Shot 2023-05-12 at 15 25 38

File: index.html [hunk 3]
```diff
@@ -189,111 +223,118 @@ <h4 class="colorh4">CANOPY</h4>
...
+                            href="https://www.linkedin.com/in/anold-mwawasi">
+                            <i class="fa fa-linkedin social"></i>
+                        </a>
+                    </li>
+                </ul>
+                <div class="getresume">
+                    <a title="get my resume" href="#">Get my resume</a>
+                </div>
+            </article>
+            <article id="what_i_do" class="padding">
```
Comment (divinecharlotte) [inline index.html:268]: - [ ] kindly consider giving this section the same margin and padding ,width and fort the `get my resume` button give it the same color as the one in Figma to implement exactly the one in the design consider checking on the below screenshots to understand what you have to update clearer
- your design 👇
Screen Shot 2023-05-12 at 15 31 51

- Figma design 👇 
Screen Shot 2023-05-12 at 15 32 59

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @MPDADDY      , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- linters are passing 👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️

- [ ] kindly consider customizing the authors section in your readme to make it contain only you as you are the one who participated in this branch and this will make it more professional

- [ ] kindly consider customizing your license section by removing the NOTE there to make your readme professional as that was given only to guide you not to be added I your readme 

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -18,65 +18,89 @@
...
+                        <a href="#">contact</a>
+                    </li>
+                </ul>
+                <div class="humberger bars">
+                    <i class="fa fa-bars"></i>
+                </div>
+            </nav>
+        </div>
     </header>
     <section class="intro">
```
Comment (nahnahdev) [inline index.html:43]: - [ ] Good job so far @MPDADDY However, just as the previous reviewer mentioned, kindly consider making sure that the `background image` for this section covers the whole section as given in the Figma design to make it an exact duplicate of the Figma design as specified in the Project Requirements. 😉 
- Your Design
Screen Shot 2023-05-12 at 5 52 47 PM

- The Figma Design
Screen Shot 2023-05-12 at 5 47 27 PM

File: index.html [hunk 5]
```diff
@@ -18,65 +18,89 @@
...
-            <div>
+            <div class="width">
+                <h2 class="colorh2">Tonic</h2>
                 <div class="canopy">
                     <h4 class="colorh4">CANOPY</h4>
                     <ul>
-                        <li>
+                        <li class="canopylist">
                             <span>
                                 <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
```
Comment (nahnahdev) [inline index.html:95]: - Just as the previous reviewer mentioned, kindly consider changing the background color of the icons between the list items so it looks similar to that of the Figma design 👍 
- Your Design
Screen Shot 2023-05-12 at 5 59 39 PM

- The Figma Design
Screen Shot 2023-05-12 at 5 59 56 PM

File: index.html [hunk 6]
```diff
@@ -189,111 +224,118 @@ <h4 class="colorh4">CANOPY</h4>
...
-        </article>
-        <article id="what_i_do" class="padding">
+        <div class="aboutmyself ">
+            <article class="padding">
+                <h1>About Myself</h1>
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
+                    work
+                    and experience! If you like what you see and have a project you need coded, don't hesitate to
+                    contact
+                    me.
```
Comment (nahnahdev) [inline index.html:236]: - [ ] Lastly, your `About section` looks great but in order to implement the `About me` content taking up lesser space than the `languages, skills and frameworks` as the previous reviewer suggested, I will suggest you style the main container of the section with` display: grid` and ensure that the `about me` part is wrapped in a separate container and takes 40% of the page and the `languages, skills and frameworks` also wrapped in a separate container and takes 60% of the page.
- This way your design looks more similar to the Figma design 😉 
- Your Design
Screen Shot 2023-05-12 at 6 20 34 PM

- The Figma Design

Screen Shot 2023-05-12 at 6 20 22 PM

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @MPDADDY , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- Most of the requested changes implemented ✔️ 
- No linter errors ✔️ 
- Good use of GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @MPDADDY 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good use CSS selectors✔️  
- Moderate UI ✅


## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by MPDADDY** [comment]
This desktop version is now good-looking.

**Pull request merged**